src/app/(main)/project/[project_id]/chat/[chat_id]/page.tsx

```diff
@@ -14,6 +15,8 @@ import { useEffect, useRef, useState } from "react";
 export default function Chat(params: { params: { project_id: string, chat_id: string } }) {
 
     const { project_id, chat_id } = params.params;
+    const projectQuery = useQuery(["project", project_id], () => API.projects.get(project_id));
+    const project: Project | undefined = projectQuery.data || undefined;
```

Comment:
@GokulramGHV no need for this, creating a new chat will result in an error anyways. No need to query an additional endpoint.